searx/engines/genius.py

```diff
@@ -27,49 +32,54 @@
 
 
 def request(query, params):
-    params['url'] = search_url.format(query=urlencode({'q': query}),
-                                      index='multi',
-                                      page_size=page_size,
-                                      pageno=params['pageno'])
+    params['url'] = search_url.format(
+        query=urlencode({'q': query}),
+        index='multi',
+        page_size=page_size,
+        pageno=params['pageno'],
+    )
     return params
 
 
 def parse_lyric(hit):
     try:
         content = hit['highlights'][0]['value']
-    except:
+    except Exception as e:  # pylint: disable=broad-except
+        logger.error(e, exc_info=True)
```

Comment:
Is the exception an error that should be fixed ?
I feel the error logging here is only useful for debugging purpose.

This alternate version doesn't raise an exception:
```python
highlights = hit.get('highlights')
content = highlights[0].get('value', '') if highlights else ''
```

With logging, it would be only be `logger.debug` not `logger.error` :
```python
highlights = hit.get('highlights')
content = highlights[0].get('value', '') if highlights else ''
if content == '':
  logger.debug('content is empty')
```

Reply:
This commit is about lint (see my https://github.com/searxng/searxng/issues/102#issuecomment-849640425) .. I don't want to change it functional.